apps/server/src/modules/tool/external-tool/mapper/external-tool-response.mapper.spec.ts

```diff
@@ -28,6 +23,8 @@ import {
 	ToolConfigurationEntryResponse,
 	ToolConfigurationListResponse,
 } from '../controller/dto';
+import { BasicToolConfigDO, ExternalToolDO, Lti11ToolConfigDO, Oauth2ToolConfigDO } from '../domainobject';
+import { CustomParameterDO } from '../../common/domainobject';
```

Comment:
Can you please change the folder name from domainobject to "domain" ?
It should be easy going with refactoring function of you ide without checking all imports